packages/terra-status-view/src/StatusView.scss

```diff
@@ -0,0 +1,101 @@
+@import '~terra-mixins';
+
+:local {
+  .status-view {
+    height: 100%;
+    padding-bottom: var(--terra-status-view-padding-bottom, 0.714rem);
+    padding-top: var(--terra-status-view-padding-top, 0.714rem);
+  }
+
+  .status-view-inner {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    width: 100%;
+  }
+
+  .glyph {
+    flex: 0 0 auto;
+    padding-bottom: 0.714rem;
```

Comment:
@kschuste - just need to add a theme-able variable for the glyph to take a separate color from the text.
For the default/terra-opensource-theme the color is: `rgba(134,138,140,0.15)`

Also validating if the icon for the error case needs to also take a different/special color (red) - was a late mention that I am checking on.

Reply:
updated to use the correct color, was using the hex value of the rgba before but using rgba directly now and can see the color difference

the color for the glyph is provided in the following variables:
--terra-status-view-glyph-no-data
--terra-status-view-glyph-no-matching-results
--terra-status-view-glyph-not-authorized
--terra-status-view-glyph-error

for those variables the svg code is provided and the color is determined by setting the 'fill' attribute in that svg code

Reply:
Update - no custom color needed for error icon. All icons colored the same unless a non-themeable/colorable custom icon is used.

Reply:
Current implementation looks great - will add UX validation.